pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -161,6 +163,48 @@ public boolean getIsSplitCommitEnabled() {
     return _controllerConf.getAcceptSplitCommit();
   }
 
+  /**
+   * Using the ideal state and segment metadata, return a list of {@link PartitionGroupMetadata}
+   * for latest segment of each partition group
+   */
+  public List<PartitionGroupMetadata> getCurrentPartitionGroupMetadataList(IdealState idealState,
```

Comment:
```suggestion
  public List<PartitionGroupMetadata> getLatestPartitionGroupMetadataList(IdealState idealState,
```

Reply:
This should still be currentPartitionGroupMetadata. It is only using the latest segment zk metadata, but it represents the current state of the partition groups.